package.json

```diff
@@ -3,14 +3,15 @@
   "version": "0.0.0",
   "license": "MIT",
   "scripts": {
-    "dev": "next dev --turbo",
```

Comment:
Let's keep this and please share the errors you're seeing! We'd like to fix them. Could you also try on the latest `next@canary` please?

Reply:
Sure. Here's the error:

![Screenshot 2024-03-11 105127](https://github.com/vercel/nextjs-subscription-payments/assets/75859865/8d581064-2218-466d-ac31-e2be2521e880)

I'm happy to report that it seems fixed in next@canary, though! Should I update the PR to use that, or should I wait until official release of stable 14.2.0?